slick-testkit/src/main/scala-2.13/com/typesafe/slick/testkit/util/VersionCompat.scala

```diff
@@ -0,0 +1,8 @@
+package com.typesafe.slick.testkit.util
+
+private[testkit] object VersionCompat {
+
+  // We may be able to get rid of this before 2.13.0 if https://github.com/scala/scala/pull/7983 is accepted
```

Comment:
FYI, this might not be needed if https://github.com/slick/slick/pull/2018 is merged

Reply:
removed